test_requirements.txt

```diff
@@ -2,7 +2,6 @@
 colour-runner==0.0.4
 coverage==3.7.1
 dj-database-url==0.3.0
-django==1.7.4
```

Comment:
The advantage of removing it, is to not maintain it (eg. no PR to update the django version for `requires.io` and as you said we leave it to the user to update it).

Maybe it would make sense to specify an interval for `test_requirements.txt`?